app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (
+    OutdatedVaccineError,
+    NotVaccinatedError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(visitor["name"])
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():

```

Comment:
use get method to avoid KeyError
`        if visitor.get("vaccine", {}).get("expiration_date") < datetime.date.today():
`